packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -88,6 +94,47 @@ export const transformPlotlyJsonToColumnProps = (jsonObj: any): IVerticalStacked
   };
 };
 
+// FIXME: Order of string axis ticks does not match the order in plotly json
+// TODO: Add support for custom hover card
+export const transformPlotlyJsonToHeatmapProps = (jsonObj: any): IHeatMapChartProps => {
+  const { data, layout } = jsonObj;
+  const firstData = data[0];
+  const heatmapDataPoints: IHeatMapChartDataPoint[] = [];
+  let zMin = Number.POSITIVE_INFINITY;
+  let zMax = Number.NEGATIVE_INFINITY;
```

Comment:
Heatmap can take x and y values in number/string/date format. Does plotly also support all 3.

Reply:
This comment can be taken as follow up.

Reply:
Not sure if Plotly heatmaps support date axis, but I saw some examples with numeric and string axis [here](https://plotly.com/javascript/heatmaps/). BTW, we can always check the [axis type](https://plotly.com/javascript/reference/layout/xaxis/#layout-xaxis-type) and convert the values to date objects if needed.